release-notes/4.1/4.1.0.md

```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
```

Comment:
The `Read more` link is broken cause of using the special character "`". I'd suggest using a simpler title.